sdk/eventhub/Azure.Messaging.EventHubs.Processor/tests/EventProcessorClient/EventProcessorClientTests.cs

```diff
@@ -491,6 +494,66 @@ public async Task StartAsyncStartsTheEventProcessorWhenProcessingHandlerProperti
             await mockProcessor.Object.StopProcessingAsync();
         }
 
+        /// <summary>
+        ///   Verifies functionality of the <see cref="EventProcessorClient.ProcessErrorAsync" />
+        ///   event.
+        /// </summary>
+        ///
+        [Test]
+        public async Task ProcessErrorAsyncDoesNotBlockStopping()
```

Comment:
Shouldn't we include a similar method for the stop handler?

Reply:
Probably, I handled the core scenario only due to time.   We should add these during the "fix up the tests" pass.

Reply:
Do you mind taking a note for that, since you were looking at taking on the initial test work?